### PR TITLE
Fix coveralls badge

### PR DESCRIPTION
**Pull request by ITaluone**
## IMPORTANT 

* [ ] The code complies with the [Coding Guidelines for C#](https://www.csharpcodingguidelines.com/).
* [ ] The changes are covered by unit tests which follow the Arrange-Act-Assert syntax and the naming conventions such as is used [in these tests](../tree/develop/Tests/FluentAssertions.Equivalency.Specs/MemberMatchingSpecs.cs#L51-L430).
* [ ] If the PR adds a feature or fixes a bug, please update [the release notes](../tree/develop/docs/_pages/releases.md) with a functional description that explains what the change means to consumers of this library, which are published on the [website](https://fluentassertions.com/releases).
* [ ] If the PR changes the public API the changes needs to be included by running [AcceptApiChanges.ps1](../tree/develop/AcceptApiChanges.ps1) or [AcceptApiChanges.sh](../tree/develop/AcceptApiChanges.sh).
* [ ] If the PR affects [the documentation](../tree/develop/docs/_pages), please include your changes in this pull request so the documentation will appear on the [website](https://www.fluentassertions.com/introduction).

Since the coveralls badge is not rendering correctly, I removed the optional parameter from the badge link. Seems to fix this issue..

### CODE REVIEW

**Comment by jnyrup** [comment]
The badge renders fine for me?

**Comment by ITaluone** [comment]
Hmm.. weird now it does it fine for me too 🤔

Maybe this was a caching problem..

**Comment by jnyrup** [comment]
Could be a combination of caching and the problems Coveralls had the other day: https://status.coveralls.io/

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 [![](https://img.shields.io/nuget/dt/FluentAssertions.svg?label=nuget%20downloads)](https://www.nuget.org/packages/FluentAssertions)
 [![](https://img.shields.io/librariesio/dependents/nuget/FluentAssertions.svg?label=dependent%20libraries)](https://libraries.io/nuget/FluentAssertions)
 ![](https://img.shields.io/badge/release%20strategy-githubflow-orange.svg)
-[![Coverage Status](https://coveralls.io/repos/github/fluentassertions/fluentassertions/badge.svg?branch=develop)](https://coveralls.io/github/fluentassertions/fluentassertions?branch=develop)
+[![Coverage Status](https://coveralls.io/repos/github/fluentassertions/fluentassertions/badge.svg)](https://coveralls.io/github/fluentassertions/fluentassertions)
```
Comment (jnyrup) [inline README.md:6]: By default coveralls picks the default git branch which is `develop`.

```suggestion
[![Coverage Status](https://coveralls.io/repos/github/fluentassertions/fluentassertions/badge.svg?branch=master)](https://coveralls.io/github/fluentassertions/fluentassertions?branch=master)
```

**Review by jnyrup — CHANGES_REQUESTED** [review]
I'm fine switching to displaying the numbers for master.
As _developers_ on FA we can see the current numbers by going to [coveralls](https://coveralls.io/github/fluentassertions/fluentassertions).
For _consumers_ of FA I guess it's more "correct" to show the numbers for the published nuget package.

**Review by jnyrup — APPROVED** [review]

**Pull request merged**